### PR TITLE
Add the release note for 1.7.0

### PR DESCRIPTION
**Pull request by minwoox**

### CODE REVIEW

File: site/src/pages/release-notes/1.6.0.mdx [hunk 1]
```diff
@@ -302,7 +302,7 @@ date: 2021-04-07
     'ikhoon',
     'JunoJunho',
     'kojilin',
-    'ks-kim',
+    'ks-yim',
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.6.0.mdx:305]: Could you also update `probe` -> `probepark`? That was my mistake. 😅

File: site/src/pages/release-notes/1.7.0.mdx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+## 📈 Improvements
+
+- N/A
+
+## 🛠️ Bug fixes
+
+- You no longer see a wrong `serviceName` for annotated services when <type://ServiceNaming> is used. #3468
+  - `ServiceRequestContext.config().defaultServiceName()` no longer returns `null` if a default service name
+    is specified.
+- You no longer see a ClosedStreamException when a gRPC call is cancelled by a client. #3452 #3455
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.7.0.mdx:36]:
```suggestion
- You no longer see a <type://ClosedStreamException> when a gRPC call is cancelled by a client. #3452 #3455
```

File: site/src/pages/release-notes/1.7.0.mdx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+- You no longer see `Socket.initialize()` error message failed due to a `UnsatisfiedLinkError`. #3457 #3485
+- You can now specify an absolute URI using <type://WebClient#prepare()>. #3479
+  - You can specify a query string both in a path string and via
+    <type://WebClientRequestPreparation#queryParam(String,Object)>.
+- You no longer see redundant parameters in DocService when Kotlin data class is used. #3454 #3461
```
Comment (ikhoon) [inline site/src/pages/release-notes/1.7.0.mdx:41]:
```suggestion
- You no longer see redundant parameters in <type://DocService> when Kotlin data class is used. #3454 #3461
```

**Review by ikhoon — APPROVED** [review]
Thanks! @minwoox 🙇‍♂️
Left only minor nits.

File: site/src/pages/release-notes/1.7.0.mdx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+- N/A
+
+## 🛠️ Bug fixes
+
+- You no longer see a wrong `serviceName` for annotated services when <type://ServiceNaming> is used. #3468
+  - `ServiceRequestContext.config().defaultServiceName()` no longer returns `null` if a default service name
+    is specified.
+- You no longer see a ClosedStreamException when a gRPC call is cancelled by a client. #3452 #3455
+- You no longer see `Socket.initialize()` error message failed due to a `UnsatisfiedLinkError`. #3457 #3485
```
Comment (trustin) [inline site/src/pages/release-notes/1.7.0.mdx:37]:
```suggestion
- You no longer see the `Failed to force-initialize 'io.netty.channel.unix.Socket'` error message
  when Armeria attempts to load native libraries. #3457 #3485
```

File: site/src/pages/release-notes/1.7.0.mdx [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+  `com.fasterxml.jackson.module:jackson-module-kotlin` or `com.fasterxml.jackson.module:jackson-module-scala`
+  as a dependency. #3458
+- You no longer see an `EofException` when writing to a timed-out response stream in <type://JettyService>.
+  It's discarded silently. #3460
+- You no longer see a <type://CancellationException> when an exception is raised by
+  a <type://FilteredStreamMessage>. #3449
+- You no longer see the <type://RequestTimeoutException> log message when a response stream fails due to
+  cancellation or timeout. #3459
+- A timeout exception is created only when a request or response is timed out. #3462
+- Armeria now rejects all HTTP/1 and HTTP/2 CONNECT requests explicitly. #3482
```
Comment (trustin) [inline site/src/pages/release-notes/1.7.0.mdx:55]:
```suggestion
- Armeria now rejects all CONNECT requests explicitly, except HTTP/2 CONNECT requests with
  a `:protocol` pseudo header. #3482
```

**Comment by minwoox** [comment]
All fixed. PTAL. 😄

**Review by trustin — APPROVED** [review]
Thanks a lot, @minwoox !

**Pull request merged**